<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparisonDisplay</class>
 <widget class="QDialog" name="ComparisonDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Region comparison</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="currentBox">
       <property name="title">
        <string>Current selection</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="loadedBox">
       <property name="title">
        <string>Loaded image</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="compBox">
       <property name="title">
        <string>Comparison</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Disjunction area</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Jaccard coefficient</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="disjAreaValue">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NaN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="conjAreaValue">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Conjunction area</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="JaccardValue">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NaN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ComparisonDisplay</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
